### ISSUE TITLE
Combine information from Electronic Portfolios and Physical Holdings 

### ISSUE THREAD

**Issue by laura-ake** [emory-libraries/blacklight-catalog#119]
As the product owner, I would like for the information from electronic portfolios in Alma to be combined with the physical holdings so that only one record displays showing both requesting options for electronic viewing and physical requesting. 

User Acceptance Criteria:

- [ ] Desired functionality:  An Alma bibliographic record (with an MMSID) that is attached to both physical holdings and an electronic Portfolio should appear only once in Blacklight Discovery Layer, displaying both the physical and electronic access methods to the users, and both electronic and physical characteristics should be indicated in indexing and display (facets, etc.)  

- [ ] Primo currently accomplishes this by merging ("deduping") two or more separate records with the same MMSID but different "intellectual entity" IDs that are published from Alma during the ingest process.

**Comment by AGCooper** [comment]
Alma API call for record with AVA AVE (physical and portfolio):

https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs/990005988630302486?view=full&expand=p_avail,e_avail&apikey=l7xx4a33b57a51924be9a395b62952788f5a

In Alma:

[Uploading... image.png]

**Comment by lovinscari** [comment]
I believe this is no longer relevant since this is handled through Alma iFrame.
